podoglyph/gradebook_v
# Development

#### What does this PR do?
This PR gets the Student dashboard up and running with Courses, Semesters, and Grades. It just needs a model method on the User to return the student's GPA.

#### How should this be manually tested?
Login as student and you should see Courses, Semesters, and Grades on the dashboard.

#### Any background context you want to provide?

#### Questions:

  - Do Migrations Need to be ran?
* No
  - Do Environment Variables need to be set?
* No
  - Any other deploy steps?
* No
